src/store/actions/actions-model.ts

```diff
@@ -280,10 +277,6 @@ export const setCustodianOfRecords: ActionIF = ({ commit }, custodian: OrgPerson
   commit('mutateCustodianOfRecords', custodian)
 }
 
-//
-// Registration actions
-//
```

Comment:
This section comment is a good idea but there are actions below that aren't for registrations.